File: C_Programing/Competitive/Problems_Solving/ASSIGNMENT_12/Q1.c
//WAP which accept  from user and DISPLAY of all digits in reverse order


#include<stdio.h>

void DisplayDigit(int iNo)
{

  if(iNo<0)
  {
    iNo=-iNo;
  }
  int iDigit=0 ;

while (iNo!=0)
{
   iDigit=iNo%10;//Digit remainder check and forward in printf 
   
   printf("%d\n",iDigit);//0/4/5//
   
   iNo=iNo/10;//540/54/4/5//Digit Seperate logic 

    
}
}


int main()
{
   
      int iValue=0;

    printf("Enter number  : ");
    scanf("%d",&iValue);


   DisplayDigit(iValue);
  

    return 0;
}

File: java_Programing/Competitive/Problems_Solving/ASSIGNMENT_21/Q1.java
//WAP to calculate the product of digits of a number ....
import java.util.Scanner;

class Q1 
 {
   public static void main(String A[] )
    {
      int iValue=0;
      
        Scanner sobj=new Scanner(System.in);
      
        System.out.println("Enter Number :");
        iValue=sobj.nextInt();


       Logic lobj=new Logic();

       lobj.productofDigits(iValue);
    }   
 }

 class Logic
 {
    void productofDigits(int iNo)
    {
       int iDigit = 0;
        int iProduct = 1;

        while(iNo != 0)
        {
            iDigit = iNo % 10;
            iProduct = iProduct * iDigit;
            iNo = iNo / 10;
        }
        System.out.println("Product of Digits of Number is :"+iProduct);
    }
 }
